src/components/Wallet.module.css

```diff
@@ -0,0 +1,5 @@
+@media (min-width: 768px) {
```

Comment:
I was trying to use the `md` breakpoint variable here but couldn't figure out how. Do you know if this is exposed somehow with the way bootstrap is imported?

https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

Reply:
Bootstrap is [imported as plain CSS](https://github.com/joinmarket-webui/joinmarket-webui/blob/v0.0.5/src/index.tsx#L15) at the moment. Sass mixins not available, I guess. 
I think this is okay for now – however, if you feel it is worthwhile and want [to include Bootstrap's Sass](https://react-bootstrap.github.io/getting-started/introduction/#sass) file, feel free to do so!